tox.ini

```diff
@@ -4,8 +4,11 @@ skip_missing_interpreters = True
 
 [testenv]
 install_command = pip install --process-dependency-links {opts} {packages}
-deps = .[docs,test]
 commands =
+# install documentation and testing requirements
+# NOTE: Don't use 'deps = .[docs,test]' tox option since we want Tox to install
+# the package from sdist first
+    pip install --process-dependency-links .[docs,test]
```

Comment:
I think that `master` branch should depend only on officially released packages and tests should reflect that. Or is there an use-case where we want to reference git dependency?